cli/internal/core/runer.go

```diff
@@ -98,13 +98,19 @@ func (ivkr _runer) Run(
 	)
```

Comment:
another note - the fact that this filename isn't spelled right is real annoying (runner, not runer)

Reply:
contribution welcome ; )